app/views/companies/show.html.haml

```diff
@@ -1,51 +1,49 @@
-%header.entry-header
-  %h1.entry-title= @company.name
-  .post-title-divider
+%div{ :id => "#{@company.class.name.downcase}-#{@company.id}", class: "show-item #{@company.class.name.downcase} #{@company.class.name.downcase}-#{@company.id}" }
```

Comment:
I'm wondering if it worth doing this:
```
#{@company.class.name.downcase}
```
as opposed to just using `company` in create the ID and the classes.

I suspect this is to insulate us from a change in the class for `company` instances?  In that case, I would think using the `company` string would be better for that purpose.